base/process.jl

```diff
@@ -647,11 +647,15 @@ Run a command object, constructed with backticks. Throws an error if anything go
 including the process exiting with a non-zero status.
 """
 function run(cmds::AbstractCmd, args...)
-    if isempty(cmds.exec) # Issue 19094
-        throw(ArgumentError("Empty command cannot be run"))
+    try
+        ps = spawn(cmds, spawn_opts_inherit(args...)...)
+        success(ps) ? nothing : pipeline_error(ps)
+    catch exc
+        if isa(exc, BoundsError)
```

Comment:
AFAIK This can only throw a BoundsError if cmds is empty, so this should be fine to check? Correct me if I'm wrong, Thanks!